src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -61,7 +61,6 @@ case class Profile (
     pi: String,
     nonProfitStatus: String,
     linkedNihUsername: Option[String] = None,
-    lastLinkTime: Option[Long] = None,
```

Comment:
yes that's right. `currentVersion` is used to prompt users to fill out a new version of the profile when logging in so no change needed here